ajax/libs/d3-geo/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "d3-geo",
+  "filename": "d3-geo.min.js",
+  "description": "Shapes and calculators for spherical coordinates.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "geo",
+    "maps",
+    "cartography"
+  ],
+  "homepage": "https://d3js.org/d3-geo/",
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Mike Bostock",
+    "url": "https://bost.ocks.org/mike"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d3/d3-geo.git"
+  },
+  "npmName": "d3-geo",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
```

Comment:
It will fetch `package.js` in v0.0.1 and some `png` files in v0.0.4. Let's exclude them using `"**/!(*.png|package.js)"`. Thanks.